api/Makefile

```diff
@@ -1,3 +1,12 @@
+AIR_BIN=bin/air
```

Comment:
Wait, should it be installed inside the repo? not globally?

Reply:
@plutov , I just wanted to give option to developers in case they do not know about air or it's setup, they can directly run formulosity without having to worry about setting up AIR

Reply:
If they don't know about air then they probably don't need it.

Reply:
haha, makes sense, will make the changes and let you know